group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupMetadataManager.java

```diff
@@ -3791,6 +3782,7 @@ public void onLoaded() {
                     break;
             }
         });
+        scheduleClassicGroupSizeCounter();
```

Comment:
In `onLoaded`, I wonder if we should compute the metrics while we go through the group in oder to have an initial value before we reach the first interval. What do you think?

Reply:
Should we cancel it when we unload?